lib/lte_link_control/lte_lc_helpers.h

```diff
@@ -341,6 +349,19 @@ int parse_coneval(const char *at_response, struct lte_lc_conn_eval_params *param
  */
 int parse_mdmev(const char *at_response, enum lte_lc_modem_evt *modem_evt);
 
+/* @brief Parse a RAI response and extract RAI configuration.
```

Comment:
There are several functions which talk about responses, when they are really for parsing notifications. IMO this can be merged as it is, but we could later do a cleanup for all such instances. After all, the response to `%RAI` is only `OK` or `ERROR`, only the unsolicited notification contains the configuration.

Reply:
Yep, I noticed this same thing and didn't want to do a big commenting/documentation changes. There are many such small things I'd like to change in lte_lc comments.